### PR TITLE
Ff checklists

### PR DESCRIPTION
**Pull request by FFincannon**
Contains Luke's WebAuthn Readiness Checklist and Karen's Integration Review Standard

### CODE REVIEW

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+* Set userVerification to “discouraged” in the PublicKeyCredentialCreationOptions.
+
+  _Note: if the PIN has already been set, the user might be prompted to enter a PIN_
+
+* Set userVerification to “discouraged” in the PublicKeyCredentialRequestOptions
+
+
+=== Attestation
+
+The Attestation Object is applicable only to CTAP2 and WebAuthn implementations. Refer to the https://www.w3.org/TR/webauthn/#sctn-attestation[W3C standard] to learn more. We recommend always saving the raw attestation statement after every credential creation.
```
Comment (emlun) [inline content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: U2F (CTAP1) indeed doesn't have the same attestation object, but it does provide an attestation certificate that's equally relevant to store.

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+* Gracefully recover if a YubiKey is not touched for registration
+* Authenticate using YubiKey (CTAP1/U2F enabled)
+* Authenticate using YubiKey (CTAP2/FIDO2 enabled)
+* Prompt to insert YubiKey as appropriate for authentication
+* Prompt to touch YubiKey as appropriate for authentication
+* Gracefully recover if a YubiKey is not presented for authentication
+* Gracefully recover if a YubiKey is not touched for authentication
+* Unregister/remove a YubiKey (self-service or admin request)
+* Name or rename registered YubiKey
+* Prevent the same user from registering the same YubiKey multiple times
```
Comment (emlun) [inline content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: Stray thought: is it sufficiently clear that this is about having the same YubiKey registered with multiple credentials at the same time - as opposed to having it registered, then de-registering it and then registering it again?

File: content/WebAuthn/WebAuthn_Developer_Guide/Overview.adoc [hunk 3]
```diff
@@ -1,92 +1,56 @@
...
+=== Building a WebAuthn Experience
 
-=== Relying Party
-The Relying Party's environment consists conceptually of at least a web server and the server-side portions of a web application, plus a WebAuthn server. The WebAuthn server has a trust store, containing the (public) trust anchors for the attestation of FIDO2 Authenticators. Note: a trust store is needed only if RP cares about attestation metadata.
+The Yubico Developer Program provides workshops, documentation, implementation guides, APIs, and SDKs. To get started,
 
+1. Sign up to the Yubico Developer Program to access the resources for implementing the FIDO2 and Web Authentication specifications.
+2. Watch FIDO2 / Web Authentication webinars to learn more about the core concepts and how to integrate WebAuthn RP operations into an authentication server.
+4. Read the developer guide for an introduction to WebAuthn/FIDO2 using Yubico’s Java WebAuthn Server.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply content/WebAuthn/WebAuthn_Developer_Guide/Overview.adoc]: Oops.

File: content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+For second factor flows, to prevent a PIN prompt when using a YubiKey for authentication, we recommended setting UV to ``discouraged``.
+
+1. What values are set for UV for registration?
+2. What values are set for UV for authentication?
+
+
+===== Saving the Attestation Object
+
+The Attestation Object is applicable only to CTAP2 and WebAuthn implementations.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc]: Emil, will you please put that information in, then, in the requisite form?

File: content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+===== Restrictions
+
+As a Relying Party, are you implementing any restrictions on the use of security keys such as checking the user agent?
+
+
+==== Authenticator Lifecycle
+
+The following YubiKey lifecycle items will be reviewed. Not all of these items will apply to every implementation. See the [Best Practices] section for more detailed information.
+
+1. How many YubiKeys per account can be registered? Recommendation is at least two: primary key and a back-up.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc]: ...well, "at least two" does indicate that one can have more than two keys. Maybe we could say "best practice is at least two and the recommendation is more".... but if I were reading that I would want to know how many more, and I would be somewhat concerned that if I had multiple keys, I would leave them lying around. However, I am not the target audience. If you think there should be no upper limit, please put that in.

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc (hunk 1 above)
Comment (FFincannon) [reply content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: Could you please add in this information?

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc (hunk 2 above)
Comment (FFincannon) [reply content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: No, I don't think it is clear enough. Please will you change it to read "* Prevent a user from registering the same YubiKey with multiple credentials at the same time, as opposed to registering the YubiKey, deregistering it, then re-registering it ad infinitum, which is permissible."

File: content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (emlun) [reply content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc]: Wait... never mind! I forgot that CTAP1 responses are also packaged into the WebAuthn attestation object, so no change is needed here.

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc (hunk 1 above)
Comment (emlun) [reply content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: Sorry, never mind! I forgot that the CTAP1 registration response is also packaged into the WebAuthn attestation object, so no change is needed here.

File: content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (emlun) [reply content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc]: Yes, that is definitely a legitimate concern if we're thinking only about YubiKeys - but it's less of an issue for platform credentials, and platform authenticators wouldn't be very useful if RPs limit the number of credentials to max 2 per user.

[the remark this answers is not in the document]
Comment (emlun) [reply content/WebAuthn/WebAuthn_Developer_Guide/integration-review-standard-fido.adoc]: Opened meta-pull-request: FFincannon/developers.yubico.com#1

File: content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc (hunk 2 above)
Comment (emlun) [reply content/WebAuthn/WebAuthn_Developer_Guide/webauthn-readiness-checklist.adoc]: Opened meta-pull-request: FFincannon/developers.yubico.com#1

**Pull request merged**